content/en/continuous_integration/explore_pipelines.md

```diff
@@ -13,15 +13,15 @@ Your pipelines appear in the [Pipelines][1] page under the CI menu.
 
 The Pipelines page shows aggregate stats for the default branch of each pipeline over the selected time frame, as well as the status of the latest pipeline execution. Use this page to see all your pipelines and get a quick view of their health. The Pipelines page shows metrics for the _default_ branch, usually named something like `main` or `prod`. 
 
-Metrics shown include build frequency, failure rate, average duration, and 95th percentile duration. This information reveals which pipelines are high-usage and potentially high resource consumers. The last build result, duration, and last run time show you the the effect of the last commit.
+Metrics shown include build frequency, failure rate, average duration, and 95th percentile duration. This information reveals which pipelines are high-usage and potentially high resource consumers. The last build result, duration, and last runtime show you the the effect of the last commit.
```

Comment:
```suggestion
Metrics shown include build frequency, failure rate, average duration, and 95th percentile duration. This information reveals which pipelines are high-usage and potentially high resource consumers. The last build result, duration, and last run time show you the the effect of the last commit.
```
This should be "last run time" because it's the time of the last run (not the last runtime)